src/features/authentication/reducers/loggedinUserSlice.ts

```diff
@@ -14,15 +17,43 @@ const userSlice = createSlice({
     initialState,
     reducers: {
         login: (state, action) => {
+            if (!action.payload.favoriteHeadLines) action.payload.favoriteHeadLineIds = []
+            if (!action.payload.notifications) action.payload.notifications = []
             state.loggedinUser = action.payload
         },
         logout: state => {
             state.loggedinUser = null
+        },
+        addFavoriteHeadline: (state, action) => {
```

Comment:
there is a type to action and payload, it looks like this

`(state, action: PayloadAction<Notification>)`

Reply:
import PayloadAction from redux